### PR TITLE
Rockchip: RK3399: linux.aarch64.conf: enable PCIe and SATA

### PR DESCRIPTION
**Pull request by lkundrak**
Hi,

Rockpro64 optionally ships with a storage case [1] intended to host a PCIe SATA controller and hard drives. This is useful for a media library -- let's enable it.
    
[1] https://wiki.pine64.org/index.php/NASCase

Tested with an actual PCIe-SATA adapter and a SATA drive.

Cc: @petr-chlupaty

Thanks
Lubo

### CODE REVIEW

File: projects/Rockchip/devices/RK3399/linux/default/linux.aarch64.conf [hunk 1]
```diff
@@ -2977,6 +3306,7 @@ CONFIG_VIDEO_EM28XX_V4L2=m
 CONFIG_VIDEO_EM28XX_ALSA=m
 CONFIG_VIDEO_EM28XX_DVB=m
 CONFIG_VIDEO_EM28XX_RC=m
+# CONFIG_MEDIA_PCI_SUPPORT is not set
```
Comment (knaerzche) [inline projects/Rockchip/devices/RK3399/linux/default/linux.aarch64.conf:3405]: While your at that: could you please set this to "y" and select the PCI(e) DVB drivers as module (I planed to do this for RK3399 anyways)

Comment (lkundrak) [reply projects/Rockchip/devices/RK3399/linux/default/linux.aarch64.conf:3405]: That sounds like a good idea. I'll submit an updated branch shortly.
Note that I can only compile-test that.

**Comment by CvH** [comment]
@knaerzche pls review

**Review by knaerzche — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**